model.py

```diff
@@ -2842,11 +2842,6 @@ def calculate_presentation(self, policy=None):
         ):
             changed = True
 
-        if changed:
-            # last_update_time tracks the last time the data 
-            # actually changed.
-            self.last_update_time = datetime.datetime.utcnow()
```

Comment:
It turns out Edition doesn't have a .last_update_time at all! This code is setting a variable on the Edition object that is not stored in the database or used anywhere.